flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/NettyShuffleEnvironmentConfiguration.java

```diff
@@ -60,6 +60,8 @@
 
 	private final boolean isNetworkDetailedMetrics;
 
+	private final boolean isReleaseExternallyManagedPartitionsOnConsumption;
```

Comment:
If we renamed the field `isManagedExternally` in the `ResultPartitionDeploymentDescriptor` into `releaseOnConsumption`, we could rename this field into `forcePartitionReleaseOnConsumption`.